sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -44,7 +50,11 @@ import org.apache.spark.sql.catalyst.plans.logical.{LogicalPlan, Project}
  *                 unless they are specified with full qualified table name with database prefix.
  */
 case class CreateViewCommand(
-    tableDesc: CatalogTable,
+    name: TableIdentifier,
+    userSpecifiedColumns: Seq[(String, Option[String])],
+    comment: Option[String],
+    properties: Map[String, String],
+    originalText: Option[String],
     child: LogicalPlan,
```

Comment:
I think it's OK that `CreateViewCommand` shows the original parsed logical plan in its physical plan tree since this logical plan is just an argument of the command, it's not executed when creating the view anyway.